src/main/io/osd.c

```diff
@@ -619,12 +392,17 @@ void osdInit(void)
 
     sprintf(string_buffer, "BF VERSION: %s", FC_VERSION_STRING);
     max7456Write(5, 6, string_buffer);
-    max7456Write(7, 7, "MENU: THRT MID");
-    max7456Write(13, 8, "YAW RIGHT");
-    max7456Write(13, 9, "PITCH UP");
+    max7456Write(7, 7,  STARTUP_HELP_TEXT1);
```

Comment:
Why is `max7456Write` being called directly? This should be done by using the `displayPort`